docs/io/ascii/read.rst

```diff
@@ -23,6 +23,7 @@ format, for example::
    >>> data = astropy.io.ascii.read('t/nls1_stackinfo.dbout', data_start=2, delimiter='|')  # doctest: +SKIP
    >>> data = astropy.io.ascii.read('t/simple.txt', quotechar="'")  # doctest: +SKIP
    >>> data = astropy.io.ascii.read('t/simple4.txt', format='no_header', delimiter='|')  # doctest: +SKIP
+   >>> data = astropy.io.ascii.read('t/tab_and_space.txt', delimiter='\s')  # doctest: +SKIP
```

Comment:
But in Python 3.6 `'\s'` raises an DeprecationWarning because it's not a valid escape sequence. That's what I meant.

The valid escape sequences are listed in https://docs.python.org/3.6/reference/lexical_analysis.html#string-and-bytes-literals.